users/tests/test_admin_update_stats.py

```
@@ -0,0 +1,149 @@
+        # requested_by 는 auth.User email 이 users.User 에 없으면 None
+        assert all(
+            r.requested_by_id is None
+            for r in StatsRefreshRequest.objects.all()
+        )
+        admin_instance.message_user.assert_called_once()
+
+    def test_rejects_selection_over_limit(self, superuser, db, stub_redis):
+        users = [
+            User.objects.create(
+                velog_uuid=uuid.uuid4(),
+                access_token="t",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "access_token"

(S106)